pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -545,7 +542,7 @@ metadata:
 spec:
   meta: |
     id: createputsvc
-    runNamespace: {{ .Volume.runNamespace }}
```

Comment:
JivaRunNamespace is not defined by the time this runtask gets executed.. you need to add this in `jiva-volume-create-default` CAS template as well

Reply:
JivaRunNamespace will be defined while calling `jiva-volume-create-default`, it is already there. Please verify it again.